src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader/Loader';
+import { Person } from '../../types';
+import { PeopleTable } from '../peopleTable/peopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const preparedPeople = (peopleData: Person[]) => {
```

Comment:
```suggestion
  const preparePeople = (peopleData: Person[]) => {
```
You can move this function to utils/helpers file/folder